/**:
  ros__parameters:
    body_length: 0.42          # From robot_physical_properties.xacro
    body_width: 0.17           # From robot_physical_properties.xacro  
    wheel_radius: 0.05         # From robot_physical_properties.xacro
    wheel_length: 0.045        # From robot_physical_properties.xacro
    wheel_connector_length: 0.1 # From robot_physical_properties.xacro
    max_steering_angle: 0.5236  # 30 degrees in radians
    max_velocity: 2.0           # m/s